app/src/editor/shared/components/ModuleHeader/index.jsx

```diff
@@ -26,37 +27,41 @@ const ModuleHeader = ({
     const [editing, setEditing] = useState(false)
 
     return (
-        <div
-            className={cx(styles.root, className)}
-            {...props}
-        >
-            {/* TODO: Replace the following line with the actual toggle. This here is just a placeholder. */}
-            <div className={styles.expandToggle} />
+        <Fragment>
+            {/* ModuleHeader's minWidth is always 92px. */}
```

Comment:
I wonder where this magic number comes 🤔

Reply:
Explained it in 4da8eaf.